src/governor.cairo

```diff
@@ -165,22 +194,27 @@ pub mod Governor {
         Canceled: Canceled,
         Executed: Executed,
         Reconfigured: Reconfigured,
+        StakerAdded: StakerAdded,
+        StakerRemoved: StakerRemoved,
     }
 
     #[storage]
     struct Storage {
         staker: IStakerDispatcher,
+        allowed_stakers: Map<ContractAddress, bool>,
         config_versions: Map<u64, Config>,
         latest_config_version: u64,
         nonce: u64,
         proposals: Map<felt252, ProposalInfo>,
         latest_proposal_by_proposer: Map<ContractAddress, felt252>,
-        vote: Map<(felt252, ContractAddress), u8>,
+        vote: Map<(felt252, ContractAddress, ContractAddress), u8>,
```

Comment:
is this change to storage backwards compatible? note when upgrading the contract, the existing storage values are not migrated, so self.vote.read for the default staker would not read the value from before the contract upgrade